### ISSUE TITLE
VDP Versioning as it applies-to security researchers

### ISSUE THREAD

**Issue by chudel** [ombegov/policy-v2#231]
Perhaps an "implementation detail" too specific for the policy, but I would like to see:
 - A note that good-faith submissions will be treated as such under the VDP that is in-effect at the time of submission.  It would be worrisome if a vulnerability was submitted and then the VDP modified in a way that made future submissions of that type out-of-scope or inappropriate.  Security Researchers will most appreciate not being "caught off-guard" by having a risk of the policy changing and being subject to the new/updated rules.

 - VDPs available in electronic form should be digitally signed so researchers and agencies alike can ensure that the VDP is authentic

 - VDP Version History should be made available.

**Comment by zmanion** [comment]
Agree with all three, is HTTPS sufficient for the second bullet?

**Comment by chudel** [comment]
Yes, https is acceptable for the transport, but I would still like to see the VDP digitally signed with an agency private key so anyone can use the agency’s public key to verify the authenticity / integrity of what’s written.
